anagrafica/management/commands/valorizza_email_servizio.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+from django.core.management.base import BaseCommand
+
+from anagrafica.models import Persona
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument('--dryrun', type=bool, default=False)
+
+    def handle(self, **options):
+        self.stdout.write(self.style.SUCCESS("Valuto email di servizio gia' presenti..."))
+        persone = Persona.objects.all()
```

Comment:
Il numero di persone in database e' molto alto, non e' realistico fare un loop su tutte quante. Suggerirei di trasformare la logica per determinare se un volontario ha una email di servizio o meno in un oggetto che puo' essere utilizzato per fare query con l'ORM. 

Usiamo `django-conceptq` per facilitare questo: e' possibile creare un singolo metodo `query_ha_email_servizio`, qualcosa come:
```python
@staticmethod
@concept
def query_ha_email_servizio(cls):
    # FIXME
    suffisso = "@cri.it"
    return Q(email_contatto__endswith=suffisso) |
           Q(utenza__email__endswith=suffisso) |
           Q(email_servizio__isnull=False)
```

Questo puo' quindi essere utilizzato per fare query, utilizzando `Persona.query_*().q` come oggetto `Q`, es.:
```python
@property
def ha_email_servizio(self):
    return Persona.objects.filter(pk=self.pk, Persona.query_ha_email_servizio().q).exists()
```

Oppure utilizzando il metodo per ottenere direttamente un queryset, es.:
```python
persone = Persona.query_ha_email_servizio()
for persona in persone:
    assert persona.ha_email_servizio
    pass
```

Vedi:
- Un esempio di concept gia' presente, [Persona.query_eta_minima](https://github.com/CroceRossaItaliana/jorvik/blob/master/anagrafica/models.py#L434)
- [django-conceptq](https://pypi.python.org/pypi/django-conceptq)